Cargo.toml

```diff
@@ -17,9 +17,14 @@ categories = [ "no-std", "rust-patterns" ]
 version = "0.4"
 optional = true
 
+[dependencies.compiletest_rs]
```

Comment:
Could this be a `dev-dependency` instead?

Reply:
I'm afraid it can't at the moment - rust-lang/cargo#1596 and it needs to be optional, because, due to language features, it simply doesn't build on stable/beta

Reply:
Really